kfs-core/src/main/java/edu/arizona/kfs/gl/batch/BudgetAdjustmentGenerationStep.java

```diff
@@ -0,0 +1,27 @@
+package edu.arizona.kfs.gl.batch;
+
+import org.kuali.kfs.sys.batch.AbstractWrappedBatchStep;
+import org.kuali.kfs.sys.batch.service.WrappedBatchExecutorService.CustomBatchExecutor;
+
+import edu.arizona.kfs.gl.batch.service.BudgetAdjustmentCashTransferService;
+
+/**
+ * A step to generate a file of budget adjustment cash transfer entries, that can then be scrubbed and posted.
+ */
+public class BudgetAdjustmentGenerationStep extends AbstractWrappedBatchStep {
+	protected BudgetAdjustmentCashTransferService budgetAdjustmentCashTransferService;
+
+    @Override
+    protected CustomBatchExecutor getCustomBatchExecutor() {
+        return new CustomBatchExecutor() {
+            public boolean execute() {
+            	budgetAdjustmentCashTransferService.generateBudgetAdjustmentCashTransferTransactions();
```

Comment:
returns true, no matter what happens inside the budgetAdjustmentCashTransferService... ?